src/main/java/com/puppycrawl/tools/checkstyle/checks/javadoc/SummaryJavadocCheck.java

```diff
@@ -84,22 +92,22 @@
     /** Period literal. */
     private static final String PERIOD = ".";
 
-    /**
-     * Stores allowed values in document for inherit doc literal.
-     */
-    private static final Set<Integer> SKIP_TOKENS = new HashSet<>(
-        Arrays.asList(JavadocTokenTypes.NEWLINE,
-                       JavadocTokenTypes.LEADING_ASTERISK,
-                       JavadocTokenTypes.EOF)
+    /** Inherit doc literal. */
+    private static final String INHERIT_DOC = "{@inheritDoc}";
```

Comment:
@sagar-shah94 One example in your current code is: https://github.com/checkstyle/checkstyle/pull/4047/files#diff-0bacc6c5fac536bdd9bde091d0bb53beR171

This will be part of new issue.